### ISSUE TITLE
Where are the latest commits?

### ISSUE THREAD

**Issue by Robindfuller** [electron-userland/electron-prebuilt-compile#91]
NPM has version 8.2.0 at only 5 months old but this repo hasn't had a commit since December 2018? Where may I find the latest version?

**Comment by smee30** [comment]
Right! Please push the latest commits - resolving problems without them is hard.
